changebot/github/tests/test_github_api.py

```diff
@@ -131,15 +131,24 @@ def test_has_modified(self):
             assert not self.pr.has_modified(['notthis.txt'])
 
 
-@patch('time.gmtime')
+@patch('changebot.github.github_api.datetime')
```

Comment:
Note to self: http://www.voidspace.org.uk/python/mock/examples.html#partial-mocking